app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -57,85 +51,100 @@ import org.mozilla.fenix.wallpapers.Wallpaper
-                            message = "", // overwritten by WallpaperSnackbar
-                            duration = SnackbarDuration.Short,
-                        )
-                    }
-                    onSelectWallpaper(updatedWallpaper)
-                },
-            )
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
+                .verticalScroll(rememberScrollState()),
+        ) {
+            wallpaperGroups.forEach { (collection, wallpapers) ->
+                if (wallpapers.isNotEmpty()) {
+                    WallpaperGroupHeading(
+                        collection,
+                        onLearnMoreClick,
+                    )
+                    WallpaperThumbnails(
+                        wallpapers = wallpapers,
+                        defaultWallpaper = defaultWallpaper,
+                        loadWallpaperResource = loadWallpaperResource,
+                        selectedWallpaper = selectedWallpaper,
+                        onSelectWallpaper = onSelectWallpaper,
+                    )
+                }
+            }
         }
     }
 }
 
 @Composable
-private fun WallpaperSnackbar(
-    onViewWallpaper: () -> Unit,
+private fun WallpaperGroupHeading(
+    collection: Wallpaper.Collection,
+    onLearnMoreClick: (String) -> Unit,
 ) {
-    Snackbar(
-        modifier = Modifier
-            .padding(8.dp)
-            .heightIn(min = 48.dp),
-        backgroundColor = FirefoxTheme.colors.actionPrimary,
-        content = {
+    Column(
+        modifier = Modifier.padding(
+            top = 16.dp,
+            start = 18.dp,
+            end = 18.dp,
+            bottom = 12.dp,
+        ),
+    ) {
+        if (collection.name != "classic-firefox") {
             Text(
-                modifier = Modifier.padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
-                text = stringResource(R.string.wallpaper_updated_snackbar_message),
-                textAlign = TextAlign.Start,
-                color = FirefoxTheme.colors.textOnColorPrimary,
-                overflow = TextOverflow.Ellipsis,
-                maxLines = 2,
-                style = FirefoxTheme.typography.headline7,
+                text = stringResource(R.string.seasonal_wallpaper_title),
+                modifier = Modifier.padding(bottom = 2.dp),
+                color = FirefoxTheme.colors.textSecondary,
+                style = FirefoxTheme.typography.subtitle2,
             )
-        },
-        action = {
-            TextButton(
-                text = stringResource(R.string.wallpaper_updated_snackbar_action),
-                onClick = onViewWallpaper,
-                modifier = Modifier.padding(all = 8.dp),
-                textColor = FirefoxTheme.colors.textOnColorPrimary,
+
+            Row(modifier = Modifier.fillMaxWidth()) {
+                Text(
+                    text = stringResource(R.string.seasonal_wallpaper_description),
```

Comment:
nit: Would you leave a comment about these string resource that reads something like:

```
// Since the last new collection of wallpapers was tied directly to an MR release, it was decided that we should use string resources for these titles and descriptions so they could be localized. In the future, we may want to either use the dynamic wallpaper properties with localized fallbacks or invest in a method of localizing the remote strings themselves. 
```